www/js/lib/uiUtil.js

```diff
@@ -940,7 +940,11 @@ function showReturnLink () {
 // Function to switch back to currently loaded page
 function returnToCurrentPage () {
     document.getElementById('liConfigureNav').classList.remove('active');
+    document.getElementById('liAboutNav').classList.remove('active');
     document.getElementById('liHomeNav').classList.add('active');
+
+    document.getElementById('btnHome').focus();
+
```

Comment:
I don't think you need the white space on lines 945 and 947, just because it's not conceptually an action that needs highlighting in some way.

Reply:
Will remove